### PR TITLE
package solution for the Microsoft.Commerce.UsageAggregation API

### PR DESCRIPTION
**Pull request by Matt-Westphal**
This is the package from the generated code that came from the Hydra output.  The corresponding pull request is in progress as well.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@Matt-Westphal__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by Matt-Westphal** [comment]
How does AZPRBot know if I'm a MS employee or not.  I am, but it doesn't appear to recognize that fact

File: src/ResourceManagement/Commerce/UsageAggregates/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("UsageAggregates")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
```
Comment (yugangw-msft) [inline src/ResourceManagement/Commerce/UsageAggregates/Properties/AssemblyInfo.cs]: you might need to fill in company information like other library projects do

Comment (Matt-Westphal) [inline src/ResourceManagement/Commerce/UsageAggregates/Properties/AssemblyInfo.cs]: Thanks I missed this when going through the first time, I've made the updates and sync'd

**Comment by ogail** [comment]
@azuresdkci add to whitelist

**Comment by ogail** [comment]
@azuresdkci retest this please

**Pull request merged**